### PR TITLE
Initial test

### PR DESCRIPTION
**Pull request by bszirtes**
Basic implementation of two test cases where, in the first case, the client sends a file descriptor to the server by URL, and in the second case, it's the other way around.

### CODE REVIEW

File: test/proto/grpcfd_test.proto [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+option go_package = "github.com/edwarnicke/grpcfd/test/proto";
+
+message FD {
+    string name = 1;
+}
+
+service FDService {
+    rpc ServerReceive (FD) returns (FD);
+    rpc ServerSend (FD) returns (FD);
+}
```
Comment (denis-tingaikin) [inline test/proto/grpcfd_test.proto:13]: Can we use grpc health service (google.golang.org/grpc/health/grpc_health_v1) instead of creating own proto service?

**Review by denis-tingaikin — COMMENTED** [review]
Also, I think we can have a group of tests for grpcfd covering. One of them totally requires running a service with grpc on localhost, as presented in this PR.

The second group can be just simple unit tests to protect against corner cases (no nil return if error is nil, and vice versa).


Anyway, this is looking like a good first step. Only recomennded to get rid of adding dep on proto with using default health service defenition from the grpc health package.

File: test/proto/grpcfd_test.proto (hunk 1 above)
Comment (bszirtes) [reply test/proto/grpcfd_test.proto:13]: We discussed this matter, and I will try using the mentioned proto service. Thank you for the suggestion.

**Review by denis-tingaikin — COMMENTED** [review]
LGTM.

After linter is fixed it can be merged.
